src/evaluate.ts

```diff
@@ -34,6 +33,10 @@ function interruptAllEvaluations() {
         } else {
             vscode.window.showInformationMessage('Interruption command finished (unknown results)');
         }
+        if (outputWindow.getEditQueueLength() > 0) {
+            outputWindow.clearEditQueue();
+            outputWindow.appendPrompt();
```

Comment:
Maybe the edit queue shouldn't be exported from the outputWindow module? Seems like it is an internal concern.

Reply:
This is why I didn't export the actual queue, only a couple of functions around it. I'd say in some cases, like this, it is a concern of calling code, as the interrupt function needs to know if clearing the queue and appending a prompt is necessary. I'm open to suggestions though if you feel this code should be refactored.